client/tests/integration/roles.rs

```diff
@@ -83,7 +83,7 @@ fn register_and_grant_role_for_metadata_access() -> Result<()> {
     let set_key_value = SetKeyValue::account(
         mouse_id,
         Name::from_str("key").expect("Valid"),
-        "value".to_owned(),
+        JsonString::from_str("value")?,
```

Comment:
```suggestion
        "value".parse()?,
```

Reply:
Is it really necessary? I mean, I can implement From<&str>, but why if we have FromStr implementation? Plus imo current realization is more explicit.

Reply:
You have it already implemented. It's just a different form. You used it elsewhere

Reply:
It doesn't work with an explicit type definition. So I see no difference what to use: either `"s".parse<JsonString>()? or `JsonString::from_str("s")`
![image](https://github.com/hyperledger/iroha/assets/68690788/cef28051-76a4-48ce-a1b7-32fc8474d637)

Reply:
> So I see no difference what to use: either `"s".parse<JsonString>()?` or `JsonString::from_str("s")`

The difference is that for the 1st case you don't need to bring in `FromStr` trait into the scope. Since our tests are also our documentation this makes it simpler

Reply:
> > So I see no difference what to use: either `"s".parse<JsonString>()?` or `JsonString::from_str("s")`
> 
> The difference is that for the 1st case you don't need to bring in `FromStr` trait into the scope. Since our tests are also our documentation this makes it simpler

Haven't known that, thanks!